### ISSUE TITLE
Certificate error on the website

### ISSUE THREAD

**Issue by N7Normandy** [rkaczorek/astroberry-server#222]
Hello everyone. There is a certificate error on the website that expired on 08/31/23. As a result, the repo can no longer be updated. Untrustworthy. Is the repo no longer available? Thanks in advance :)

Error:
Certificate verification failed: The certificate is NOT trusted. The certificate chain uses expired certificate.  Could not handshake: Error in the certificate verification. [IP: 46.105.204.23 443]

**Comment by CreasaBE** [comment]
Yep same issue here when trying to set up indi-allsky as it needs packages from astroberry. Cert has indeed expired and needs to be renewed again.
It's a LetsEncrypt certificate, so it should've been done automaticaly, but apparently it has not.

**Comment by bartczar** [comment]
You can add /etc/apt/apt.conf.d/99astroberry file with the following content:
Acquire::https::www.astroberry.io::Verify-Peer "false";
Acquire::https::www.astroberry.io::Verify-Host "false";

To get a repo key from the astroberry.io you need to use this to ignore cert issue: "wget --no-check-certificate -O - https://www.astroberry.io/repo/key"

**Comment by N7Normandy** [comment]
Yeah it works . Thanks a lot.

**Comment by CreasaBE** [comment]
> You can add /etc/apt/apt.conf.d/99astroberry file with the following content: Acquire::https::www.astroberry.io::Verify-Peer "false"; Acquire::https::www.astroberry.io::Verify-Host "false";
> 
> To get a repo key from the astroberry.io you need to use this to ignore cert issue: "wget --no-check-certificate -O - https://www.astroberry.io/repo/key"

Yes although that will work, it is not a solution to the problem.
- it is insecure
- it does not mean that the website or repo is valid and can be trusted

the website itself is untrusted too because of the certificate expiring and a lot of browsers will complain and firewalls with SSL Decryption enabled will deny access

**Comment by rkaczorek** [comment]
HI guys. This is error on the server side. For some unknown reason SSL certificates did not renewed recently.
I'm gonna manually fix this right away.

**Comment by guilleortas** [comment]
It's still happening for me on a fresh install downloaded and flashed today

**Comment by rkaczorek** [comment]
Please remember that Raspberry Pi is not (by default) equipped with real time clock, which keeps system time between reboots. It gets current time from the Internet and it takes some time, especially in case of fresh install.
Now, if you try to run update too quickly after system boot (and your system time has not been updated yet ), you will always get certificate error on any software repository. It is a system feature, not a bug.
If it happens, you can check your time running `date` command in terminal window, wait a few seconds, run the command again until it shows proper date. Only then you can run system updates.
